source/_integrations/dynalite.markdown

```diff
@@ -129,6 +129,11 @@ area:
                   required: false
                   type: float
                   default: 2.0
+                level:
+                  description: Level of the channels when the preset is selected, between 0 and 1.
+                  required: false
+                  type: float
+                  default: None
```

Comment:
If there is no default we can omit it

Reply:
ok. will remove. I wanted to be explicit that there is no level set unless explicitly set for a preset, but agree it is confusing

Reply:
done